support/troubleshooting.mdx

```diff
@@ -24,20 +24,20 @@ This section is aimed at collecting common issues users have to provide quick de
     `cargo binstall cargo-shuttle`
   </Accordion>
   <Accordion title="How can I return custom errors with shuttle?">
-    CustomError is an alias for **`anyhow::Error`**, the simplest way to output a relevant message here is by using **`anyhow::Context`** like this:
+    CustomError is an alias for `anyhow::Error`, the simplest way to output a relevant message here is by using `anyhow::Context` like this:
     
     ```rust
-        pool.execute(include_str!("../schema.sql"))
+    pool.execute(include_str!("../schema.sql"))
         .await
         .context("failed to run migrations")?;
     ```
   </Accordion>
   <Accordion title="How can I handle environment variables?">
-    Add **`shuttle-secrets`** to the dependencies for your service, and add a **`Secrets.toml`** to the root of your project with the secrets you'd like to store. Make sure to add **`Secrets.toml`** to a **`.gitignore`** to omit your secrets from version control.
+    Add `shuttle-secrets` to the dependencies for your service, and add a `Secrets.toml` to the root of your project with the secrets you'd like to store. Make sure to add `Secrets*.toml` to your `.gitignore` to omit your secrets from version control.
```

Comment:
```suggestion
    Add `shuttle-secrets` to the dependencies for your service, and add a `Secrets.toml` to the root of your project with the secrets you'd like to store. Make sure to add `Secrets.toml` to your `.gitignore` to omit your secrets from version control.
```
I think you missed this

Reply:
That is an improvement I suggest having, since `Secrets*.toml` will also match against `Secrets.dev.toml`

Reply:
That makes sense :+1: